### DISCUSSION TITLE
Set specific values in on x-axis in XYchart package

### DISCUSSION THREAD

**Discussion by sargentieri** [airbnb/visx#1584] [Q&A]
I need to display data over 12 week period starting from today's date. 

I would really like the weekly increments to display those precise dates, 12 weeks apart, on the X-Axis.

How can I achieve this? 

Thanks in advance!

**Comment by williaster** [comment]
Hi @sargentieri thanks for checking out `visx`. `@visx/xychart` inherits axis props from the `@visx/axis` package, the answer to this question can be found in this discussion answer https://github.com/airbnb/visx/discussions/1200#discussioncomment-705906

Hope that helps!

**Reply by sargentieri** [reply]
Hey @williaster thanks for getting back. I was aware that the axis does inherit props, I ended up finding a working solution by modifying the scales of the XYChart. It did take me a sec to realize the syntax was different as I was trying to pass a scale function in directly. 

 I did run into some abhorrent behavior and I believe this is due to the custom scales, as it knocks the tool tip out of sync of the data point its supposed to show (the data point to activate the tooltip is off to the left of the visualized data). In order for the custom scales to work, I have to use `Group` components, to properly offset the visualizations. It seems this visually works, however the calculated data point passed to the tool tip does not like this approach. 

have you experienced anything similar to this?

thanks,
Sal

**Comment by sargentieri** [comment]
Just an update, I over used the Group component, and also set a bad custom scale for the Y value, which led to the behavior
